news/3860.bugfix

```diff
@@ -0,0 +1,2 @@
+Fixes sort portal types alphabetically in search filter
```

Comment:
For languages that don't have an alphabet. Also we should refer to the implementation in the other package. And use `@` for credit.

```suggestion
Sort portal types in search filter according to `ReallyUserFriendlyTypes` using `unidecode` in `plone.app.vocabularies`. @rohnsha0
```

Reply:
ah... gotcha... thanks btw